contrib/coredns/policy/dns.go

```diff
@@ -180,3 +198,52 @@ func ip2rr(ip net.IP, name string, class uint16) dns.RR {
 	}
 	return nil
 }
+
+// find record will return record for typ if dst is in rrCodeformat and has record for typ, if it does not have will send empty.
```

Comment:
For example, `typ = TXT` , redirect strings `A=127.0.0.1` and `A=127.0.0.1;TXT=` will return empty